<launch>	
  <!-- Launch the tracking node. Modified launch file for supporting of Kinect sensor. -->
  <node pkg="visp_auto_tracker" type="visp_auto_tracker" name="visp_auto_tracker" >
    <param name="model_path" value="$(find visp_auto_tracker)/models" />
    <param name="model_name" value="pattern" />
    <param name="debug_display" value="false" />    
    <remap from="/visp_auto_tracker/camera_info" to="/camera/rgb/camera_info"/>
    <remap from="/visp_auto_tracker/image_raw" to="/camera/rgb/image_raw"/>
  </node>
</launch>
